io.github.ptitSeb.hydracastlelabyrinth.yml

```diff
@@ -0,0 +1,46 @@
+app-id: io.github.ptitSeb.hydracastlelabyrinth
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: io.github.ptitSeb.hydracastlelabyrinth.sh
+
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/GL
+    version: '23.08'
+
+modules:
+  - name: hcl
+    sources:
+      - type: git
+        url: https://github.com/ptitSeb/hydracastlelabyrinth.git
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\\d.]+)$
+      - type: script
+        dest-filename: io.github.ptitSeb.hydracastlelabyrinth.sh
+        commands:
+          - #!/bin/bash
+          - cd /app
+          - ./hcl $@
+    buildsystem: cmake
+    config-opts:
+      - -DUSE_SDL2=ON
+    post-install:
+      - install -Dm755 io.github.ptitSeb.hydracastlelabyrinth.sh /app/bin/io.github.ptitSeb.hydracastlelabyrinth.sh
+      - desktop-file-edit --set-key=Exec --set-value="flatpak run io.github.ptitSeb.hydracastlelabyrinth" /app/share/applications/io.github.ptitSeb.hydracastlelabyrinth.desktop
```

Comment:
```suggestion
```

Definitely not. This should match what's in `command` more or less the arguments substitution